go.mod

```diff
@@ -8,6 +8,8 @@ replace github.com/coreos/go-iptables => github.com/aosedge/go-iptables v0.0.0-2
 
 replace github.com/anexia-it/fsquota => github.com/aosedge/fsquota v0.0.0-20231127111317-842d831105a7
 
+replace github.com/aosedge/aos_common => github.com/mykola-kobets-epam/aos_common v0.0.0-20240905101719-1494c876bb54
```

Comment:
better make this change in a separate WIP commit

Reply:
I was going to change "vendor update commit" once aos_common updated